src/smefit/fit_manager.py

```diff
@@ -73,6 +74,10 @@
         with open(f"{self.path}/{self.name}/{file}.json", encoding="utf-8") as f:
             results = json.load(f)
 
+        # load the fit results
```

Comment:
This part is not backward compatible: the fit_result.json might not exist for old runs. We should move this to a try except that falls back to the old methodology in case it fails.

Reply:
Ok, this is an interesting point... backward compatibility is a good thing but if brought to the extreme we might end up having a code that is a bit annoying to maintain. For this specific thing, I am not in favour of keeping backward compatibility for the simple reason that this fixes a "bad" behaviour. We should on the other hand tag versions of the code, so that if you need to revisit older fits, one can in principle go back to that version of the code. What do you think?

Reply:
Ah I like that, even better! So let's keep your change as is and tag the main code before we merge. Sounds good?

Reply:
Yes, it sounds good :)